src/AddressRegistry.Producer.Snapshot.Oslo/appsettings.json

```diff
@@ -26,7 +26,7 @@
     "BootstrapServers": "localhost:29092/"
   },
 
-  "Topic": "dev.address.snapshot.oslo",
+  "AddressTopic": "dev.address.snapshot.oslo",
```

Comment:
should this be in appsettings.json?

Reply:
this is how it was done in the AddressRegistry.Producer on which we based the project, changing the topic is probably not something that needs to be configurable...